### ISSUE TITLE
Wer is very low.

### ISSUE THREAD

**Issue by nitinyadav250** [NVIDIA/NeMo#8872]
conf:

wandb_version: 1

_wandb:
  desc: null
  value:
    python_version: 3.11.8
    cli_version: 0.16.6
    framework: huggingface
    huggingface_version: 4.39.3
    is_jupyter_run: false
    is_kaggle_kernel: false
    start_time: 1712733521.0
    t:
      1:
      - 1
      - 5
      - 9
      - 11
      - 49
      - 50
      - 51
      - 53
      - 55
      - 103
      2:
      - 1
      - 5
      - 9
      - 11
      - 49
      - 50
      - 51
      - 53
      - 55
      - 103
      3:
      - 7
      - 13
      - 14
      - 23
      4: 3.11.8
      5: 0.16.6
      6: 4.39.3
      8:
      - 5
      13: linux-x86_64
    m:
    - 1: trainer/global_step
      6:
      - 3
    - 1: train_loss
      5: 1
      6:
      - 1
    - 1: learning_rate
      5: 1
      6:
      - 1
    - 1: global_step
      5: 1
      6:
      - 1
    - 1: train_backward_timing in s
      5: 1
      6:
      - 1
    - 1: train_step_timing in s
      5: 1
      6:
      - 1
    - 1: training_batch_wer
      5: 1
      6:
      - 1
    - 1: epoch
      5: 1
      6:
      - 1
    - 1: validation_step_timing in s
      5: 1
      6:
      - 1
    - 1: val_loss
      5: 1
      6:
      - 1
    - 1: val_wer
      5: 1
      6:
      - 1
    - 1: test_step_timing in s
      5: 1
      6:
      - 1
    - 1: test_loss
      5: 1
      6:
      - 1
    - 1: test_wer
      5: 1
      6:
      - 1
cfg:
  desc: null
  value:
    sample_rate: 16000
    log_prediction: true
    ctc_reduction: mean_batch
    skip_nan_grad: false
    train_ds:
      manifest_filepath: /media/dilip/LaCie/Nitin/content/drive/MyDrive/NEW/hi/train.json
      sample_rate: 16000
      batch_size: 24
      shuffle: true
      num_workers: 16
      pin_memory: true
      use_start_end_token: false
      trim_silence: false
      max_duration: 30
      min_duration: 0.1
      is_tarred: false
      tarred_audio_filepaths: null
      shuffle_n: 2048
      bucketing_strategy: synced_randomized
      bucketing_batch_size: 8
    validation_ds:
      manifest_filepath: /media/dilip/LaCie/Nitin/content/drive/MyDrive/NEW/hi/validated.json
      sample_rate: 16000
      batch_size: 1
      max_duration: 30
      shuffle: false
      num_workers: 8
      pin_memory: true
      use_start_end_token: false
    test_ds:
      manifest_filepath: /media/dilip/LaCie/Nitin/content/drive/MyDrive/NEW/hi/test.json
      sample_rate: 16000
      batch_size: 4
      shuffle: false
      num_workers: 8
      pin_memory: true
      use_start_end_token: false
    tokenizer:
      dir: /media/dilip/LaCie/Nitin/1T/tokenizer_spe_unigram_v128
      type: bpe
    preprocessor:
      _target_: nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor
      sample_rate: 16000
      normalize: per_feature
      window_size: 0.025
      window_stride: 0.01
      window: hann
      features: 80
      n_fft: 512
      log: true
      frame_splicing: 1
      dither: 1.0e-05
      pad_to: 0
      pad_value: 0.0
    spec_augment:
      _target_: nemo.collections.asr.modules.SpectrogramAugmentation
      freq_masks: 2
      time_masks: 5
      freq_width: 27
      time_width: 0.05
    encoder:
      _target_: nemo.collections.asr.modules.ConformerEncoder
      feat_in: 80
      feat_out: -1
      n_layers: 17
      d_model: 512
      subsampling: striding
      subsampling_factor: 4
      subsampling_conv_channels: -1
      ff_expansion_factor: 4
      self_attention_model: rel_pos
      n_heads: 8
      att_context_size:
      - -1
      - -1
      xscaling: true
      untie_biases: true
      pos_emb_max_len: 5000
      conv_kernel_size: 31
      conv_norm_type: batch_norm
      dropout: 0.1
      dropout_emb: 0.0
      dropout_att: 0.1
    decoder:
      _target_: nemo.collections.asr.modules.ConvASRDecoder
      feat_in: null
      num_classes: 128
      vocabulary:
      - <unk>
      - ा
      - ▁
      - र
      - ी
      - े
      - ि
      - ल
      - न
      - त
      - ्
      - क
      - ं
      - स
      - म
      - य
      - ह
      - ो
      - ु
      - ▁स
      - ▁ब
      - द
      - ▁क
      - ▁म
      - व
      - ▁प
      - ज
      - ट
      - प
      - ग
      - ▁के
      - ▁में
      - ब
      - ू
      - ▁है
      - ▁अ
      - ▁ज
      - ▁की
      - ▁द
      - ्र
      - ड
      - श
      - च
      - ▁व
      - ▁र
      - ▁ह
      - ए
      - ▁आ
      - ▁का
      - ▁ग
      - ▁को
      - ▁से
      - ने
      - ै
      - ंग
      - थ
      - ों
      - ▁च
      - ▁ल
      - ड़
      - ▁ने
      - ▁पर
      - ध
      - ▁न
      - ई
      - फ
      - ▁श
      - ्य
      - ख
      - ▁कर
      - ़
      - छ
      - ौ
      - ण
      - भ
      - इ
      - ठ
      - घ
      - ष
      - ँ
      - ओ
      - आ
      - ः
      - झ
      - i
      - a
      - e
      - o
      - r
      - l
      - p
      - t
      - s
      - w
      - f
      - ऊ
      - h
      - m
      - ’
      - n
      - k
      - c
      - अ
      - v
      - b
      - d
      - g
      - u
      - y
      - x
      - j
      - z
      - ”
      - ॅ
      - ऋ
      - ञ
      - ‘
      - ृ
      - ऐ
      - ऑ
      - ढ
      - औ
      - ॉ
      - '&'
      - ऩ
      - “
      - q
      - उ
    optim:
      name: adamw
      lr: 0.5
      betas:
      - 0.9
      - 0.98
      weight_decay: 0.001
      sched:
        name: NoamAnnealing
        d_model: 512
        warmup_steps: 10000
        warmup_ratio: null
        min_lr: 1.0e-06
    target: nemo.collections.asr.models.ctc_bpe_models.EncDecCTCModelBPE
    nemo_version:

```[tasklist]
### Tasks
- [ ] Add a draft title or issue reference here
```

**Comment by nitinyadav250** [comment]
{
  "train_loss": 8.534393310546875,
  "learning_rate": 0.00012372159108053893,
  "global_step": 5610,
  "train_backward_timing in s": 0.15601778030395508,
  "train_step_timing in s": 0.34882450103759766,
  "training_batch_wer": 0.18061673641204834,
  "epoch": 30,
  "trainer/global_step": 5610,
  "_timestamp": 1712744688.4421802,
  "_runtime": 11167.270045042038,
  "_step": 293353,
  "validation_step_timing in s": 0.02845907211303711,
  "val_loss": 6.631827354431152,
  "val_wer": 0.12062931060791016,
  "test_step_timing in s": 0.043920040130615234,
  "test_loss": 10.514272689819336,
  "test_wer": 0.17779342830181122,
  "_wandb": {
    "runtime": 11166
  }

**Comment by nitinyadav250** [comment]
i used common voice13 hindi as dataset

**Comment by nitinyadav250** [comment]
my issue is that wer for the model i used i.e conformer_large_ctc is very low .what could be the reason for such a low wer.
@titu1994 sir

**Comment by nitinyadav250** [comment]
actually i was worried about overfitting of data btw thanks sir 🙌
